website/docs/compose/podman-compose.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Podman Compose with Podman Desktop
+---
+
+## Introduction
+
+Podman Desktop is aimed at providing the Podman experience to the developers with a Graphical User Interface. Many developers who enjoy using Podman Engine to work with containers are also avid users of Podman Compose. 
```

Comment:
I intend to add a separate section for Docker Compose. This one is specifically focused on Podman Compose.

Reply:
> I intend to add a separate section for Docker Compose. This one is specifically focused on Podman Compose.

I don't think there should be a separate section because Podman Desktop is *suppose* to be engine agnostic. If we end up adding totally separate sections, then it's not really an upstream project anymore. Both Podman Compose and Docker Compose support the Compose spec, so neutral language should instead be used because both `podman-compose` and `docker-compose` can be used.

It's like how `podman` makes sure that they are 100% compatible with any `docker` cli commands. Whatever works on docker / docker compose, will work on podman and podman compose 💯